adb_graphics/utils.py

```diff
@@ -115,6 +115,34 @@ def get_func(val: str):
     fun = getattr(module, fun_name)
     return fun
 
+# pylint: disable=unused-argument
+def join_ranges(loader, node):
+
+    '''
+    Merge two or more different ranges into a single array for color bar clevs.
+
+    e.g.: in default_specs.yml, clevs for visibility can be assigned as
+
+        clevs: !join_ranges [[0, 10, 0.1], [10, 51, 1.0]]
+
+    The join_ranges method concatenates these ranges into a single array of levels.
+    This can be useful for plots where one part of the color ranges requires higher
+    resolution than the rest, while keeping the colorbar from looking squished.
+
+    Note that a "yaml.add_constructor" is required, as shown after the method.
```

Comment:
Would you mind putting a pointer (just text that says something like "go here to read more") in default specs under clevs to this file and method's docs?

Reply:
added this to the clevs for visibility in default_specs:

"see the description provided in adb_graphics/utils.py, for the join_ranges method."